### ISSUE TITLE
Show doesn't yield any metadata

### ISSUE THREAD

**Issue by VladyslavLukyanenko** [tobymao/sqlglot#2148]
```
from sqlglot import parse_one, exp

tables = parse_one('SHOW PRIMARY KEYS IN "TEST"."PUBLIC"."customers"', read="snowflake", dialect="snowflake").find_all(exp.Table)
```

Expected output:

List of tables, in this case it should be `customers`

Instead receiving an empty result

Official documentation:
https://docs.snowflake.com/en/sql-reference/sql/show-primary-keys

**Comment by GeorgeSittas** [comment]
Hey, thanks for the report, we'll look into this soon.

**Comment by tobymao** [comment]
closed by 585d0bfcbd40125492640480c890af328d8bc51f

parsed as a command, explicit parsing is out of scope, but happy to accept a PR
